src/main/java/com/worldwarofants/game/module/tutorial/TutorialView.java

```diff
@@ -0,0 +1,34 @@
+package com.worldwarofants.game.module.tutorial;
+
+import com.worldwarofants.game.arch.AbstractView;
+import com.worldwarofants.game.arch.console.ConsoleOutputHandler;
+
+public class TutorialView extends AbstractView<TutorialViewModel> {
+
+	public TutorialView(TutorialViewModel viewModel) {
+		super(viewModel);
+	}
+
+	public void renderTutorialScreenStart() {
+		String fightPrompt = String.format("* Fight! - %s ", TutorialCommandHandler.COMMAND_FIGHT);
+		String retreatPrompt = String.format("* Retreat! - %s ", TutorialCommandHandler.COMMAND_RETREAT);
+		String quitPrompt = String.format("* Retreat! - %s ", TutorialCommandHandler.COMMAND_QUIT);
+		ConsoleOutputHandler.lineBreak();
+		ConsoleOutputHandler.spell("Your colony is under attack! You are on the brink of defeat and you must fall back in order to avoid complete annihilation! ...");
```

Comment:
Usually in professional code, and according to the standards, we should never write things such as `magical numbers` or `magical strings`. You may have heard of this. It's most commonly seen with numbers. 

Like for example, you should never have a number loosely written in your code. It must always be in a constant field with a descriptive name. If you had to write a formula in code, it should not look like this:
`triangleArea = (base * height) / 2 `.

That `2` right there is considered a magical number. Because, what the hell is it? It's hard to tell with a glance. You have to think about the formula. Also, what if you had this written out in several places? And what if you wanted to change it after a few months? You would have to scroll through a 500 lines class to find a single number.

If it was put in a constant, that number would be at the top of the class, and you can change it in one place, and affect all the code that uses it. 

Here it's the same thing, except with strings. It's not as extreme of a situation, I suppose it can be left as it is. But I'm just saying, if you want to have your code be closer to being perfect.

Reply:
Gotch, updated.